common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
```

Comment:
@Thanu I don't think we need to do this. There is a separate function to deploy a generic service. In deploy_common_services we should deploy services that are common to all products. Is there any advantage of breaking it down to another sub-function?